prow/cmd/hook/main.go

```diff
@@ -33,6 +33,7 @@ import (
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plugins"
 
+	_ "k8s.io/test-infra/prow/plugins/close"
```

Comment:
There's a unit test if you forget to add it and try to add it to `plugins.yaml`. The nice thing is that if you don't want to include a plugin in the binary you can just not include it here.

(I wanted to put these lines in the `plugins` package but it makes an import cycle :P)